lib/pages/portal/view/website_view.dart

```diff
@@ -48,9 +52,14 @@ class _WebsiteViewState extends State<WebsiteView> {
   TextEditingController _descriptionRoController;
   TextEditingController _descriptionEnController;
   final RelevanceController _relevanceController = RelevanceController();
+  String _iconPath;
```

Comment:
This doesn't need to be a class attribute. Or even a variable, really. Just use `widget.website.iconPath` directly.